presto-main/src/main/java/com/facebook/presto/cost/PlanNodeStatsEstimate.java

```diff
@@ -79,14 +118,15 @@ public boolean equals(Object o)
             return false;
         }
         PlanNodeStatsEstimate that = (PlanNodeStatsEstimate) o;
-        return Objects.equals(outputRowCount, that.outputRowCount) &&
-                Objects.equals(outputSizeInBytes, that.outputSizeInBytes);
+        return Double.compare(that.outputRowCount, outputRowCount) == 0 &&
```

Comment:
Change from `Object.equals` to `Double.compare` should be part of previous commit.